### PR TITLE
Document oauth2 app requirement for 2fa in the sync client

### PR DESCRIPTION
**Pull request by settermjd**
This relates to https://github.com/owncloud/docs/issues/1882.

### CODE REVIEW

File: docs/modules/ROOT/pages/navigating.adoc [hunk 1]
```diff
@@ -89,6 +90,12 @@ Simply click the *Account* > *Add New* button on any account tab to add a new ac
 The new account will appear as a new tab in the settings dialog, where you can adjust its settings at any time.
 Use *Account* > *Remove* to delete accounts.
 
+[NOTE]
+====
+ownCloud server must have {oauth2-app-url}[the OAuth2 app] installed, configured, and enabled to use Two-Factor Authentication.
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/navigating.adoc]:
```suggestion
To use *Two-Factor Authentication* (2FA), ownCloud server must have {oauth2-app-url}[the OAuth2 app] installed, configured, and enabled.
```

I'd try to make "Two-Factor Authentication" more prominent.

Comment (settermjd) [reply docs/modules/ROOT/pages/navigating.adoc]: That's an excellent suggestion.

**Comment by settermjd** [comment]
@michaelstingl, can you help with the breaking build?

**Comment by settermjd** [comment]
Closing in favour of #7829.

**Pull request closed without merging**